vortex-array/src/stats/mod.rs

```diff
@@ -173,7 +173,7 @@ pub trait Statistics {
     /// Compute all the requested statistics (if not already present)
     /// Returns a StatsSet with the requested stats and any additional available stats
     fn compute_all(&self, stats: &[Stat]) -> VortexResult<StatsSet> {
-        let mut stats_set = self.to_set();
+        let mut stats_set = StatsSet::default();
```

Comment:
not sure why compute_all was starting with a copy of existing stats given `self.compute` will lookup the value in the cache